tests/e2e/monitoring/test_mon_log_trimming.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import pytest
+import random
+import threading
+import re
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, bugzilla
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.resources.pod import get_mon_pods
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
need to decide which tier this test belongs and update here

Reply:
Done